frontend/src/app/libs/common-components/components/button/button.component.scss

```diff
@@ -0,0 +1,62 @@
+@mixin button-hover() {
+  transition: background-color 0.4s;
+  &:hover:not([disabled="true"]) {
+    background-color: transparentize($color: white, $amount: 0.92);
+  }
+}
+
+.button {
+  padding: 13px 16px 11px 16px;
```

Comment:
seems like you trying here to center text inside button by paddings, can we use flexbox for this?